src/transformers/models/swin/__init__.py

```diff
@@ -0,0 +1,53 @@
+# flake8: noqa
+# There's no way to ignore "F401 '...' imported but unused" warnings in this
+# module, but to preserve other warnings. So, don't check this module at all.
+
+# Copyright 2022 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import TYPE_CHECKING
+
+# rely on isort to merge the imports
+from ...file_utils import _LazyModule, is_flax_available, is_tf_available, is_torch_available, is_vision_available
```

Comment:
@sgugger I've experienced myself that `make fixup` isn't saying anything about this anymore.

e.g. here, `is_flax_available` and `is_tf_available` should not be there.

Reply:
It has never said anything about that. `flake8` is disabled for all inits because it will complain all objects are defined but not used, so it's up to us to make sure there are no unnecessary imports.

Reply:
@novice03 you can remove the unnecessary imports here

Reply:
This has not been addressed before merging.